sdk/core/azure-core/src/test/java/com/azure/core/http/rest/SyncRestProxyTests.java

```diff
@@ -98,7 +98,7 @@ public void contentTypeHeaderPriorityOverBodyParamAnnotationTest() {
             .httpClient(client)
             .build();
 
-        TestInterface testInterface = SyncRestProxy.create(TestInterface.class, pipeline);
+        TestInterface testInterface = RestProxy.create(TestInterface.class, pipeline);
```

Comment:
We should change the LocalHttpClient in this test to throw in send and implement sendSync, so that we assert that proxy called right send.